dotnetv3/SQS/SQSReceiveMessagesExample/SQSReceiveMessagesExample/SQSReceiveMessagesExample.cs

```diff
@@ -0,0 +1,115 @@
+﻿// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier:  Apache-2.0
+
+/// <summary>
+/// Shows how to receive messages from an existing Amazon SQS queue.
+/// Reads a single message from the SQS queue, displays the message, then deletes it from the queue.
+/// Can be used in conjunction with SQSSendMessagesExample to see an example of message flow.
+/// See the following for more information:
+/// https://docs.aws.amazon.com/sdk-for-net/v3/developer-guide/ReceiveMessage.html
+/// 
+/// This example was created using the AWS SDK for .NET version 3.7 and .NET 5.0.
+/// </summary>
+
+// snippet-start:[sqs.dotnetv3.SQSReceiveMessagesExample.complete]
+using System;
+using System.Threading.Tasks;
+using Amazon.SQS;
+using Amazon.SQS.Model;
+
+namespace SQSReceiveMessagesExample
+{
+  class Program
+  {
+    private const int MaxMessages = 1;
+    private const int WaitTime = 2;
+    static async Task Main(string[] args)
+    {
+      // Do some checks on the command-line
+      if (args.Length == 0)
+      {
+        Console.WriteLine("\nUsage: SQSReceiveMessages queue_url");
+        Console.WriteLine("   queue_url - The URL of an existing SQS queue.");
+        return;
+      }
+      if (!args[0].StartsWith("https://sqs."))
+      {
+        Console.WriteLine("\nThe command-line argument isn't a queue URL:");
+        Console.WriteLine($"{args[0]}");
+        return;
+      }
+
+      // Create the Amazon SQS client
+      var sqsClient = new AmazonSQSClient();
+
+      // (could verify that the queue exists)
+      // Read messages from the queue and perform appropriate actions

```

Comment:
Sentences in comments should be complete, grammatically correct, and end with a period.